### PR TITLE
Introducing Graffiti : graph engine

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-unit-tests

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-cdd-overview-tests

**Comment by nplanel** [comment]
run skydive-cdd-overview-tests

**Comment by nplanel** [comment]
run python-tests

**Comment by nplanel** [comment]
run skydive-create-docker-image

**Comment by nplanel** [comment]
run skydive-cdd-overview-tests

**Comment by nplanel** [comment]
run python-tests

**Comment by nplanel** [comment]
run skydive-create-docker-image

File: flow/enhancers/graph.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/*
```
Comment (safchain) [inline flow/enhancers/graph.go]: This file has been deleted in a previous patch, need to be removed

Comment (nplanel) [reply flow/enhancers/graph.go]: good point ... fixed

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Pull request merged**